modules/ballerina-http/src/main/java/org/ballerinalang/net/http/session/Session.java

```diff
@@ -82,11 +83,18 @@
     /**
      * Get attribute name list.
      *
-     * @return name array.
+     * @return array of keys.
```

Comment:
shouldn't it be "array of attribute names"?

Reply:
Ack